hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/ResponsePacketHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.hazelcast.spi.impl.operationexecutor;
+
+import com.hazelcast.nio.Packet;
+import com.hazelcast.spi.impl.Response;
+
+/**
+ * The {@link com.hazelcast.spi.impl.operationexecutor.ResponsePacketHandler} is responsible for handling
```

Comment:
The idea is that the OperationExecutor should not care what happens because that is part of the ResponsePacketHandler implementation. The Executor provides the threading, the actual handling of the response packet is not the responsibility of the Executor.